### PR TITLE
🔀🌐🇵🇱 add pl_PL translations

### PR DESCRIPTION
**Pull request by Kemon22**

### CODE REVIEW

File: src/modules/dailyCreditsSummary/i18n/pl_PL.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+            title: 'Kompleks budynków',
+            backgroundColor: '#b5651d',
+            textColor: 'Biały',
+        },
+        fireAlarmSystemCancel: {
+            regex: /Fałszywy alarm/u,
+            titel: 'Fałszywy alarm',
+            backgroundColor: '#c80815',
+            textColor: 'Biały',
+        },
```
Comment (jxn-30) [inline src/modules/dailyCreditsSummary/i18n/pl_PL.ts:93]: that is a bit more complex and need to match the rows in https://www.operatorratunkowy.pl/credits/daily
We will do that after the first merge

File: src/modules/dashboard/i18n/pl_PL.json [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        },
+        "type": "Typ",
+        "unavailable": "W budowie"
+    },
+    "building-types": {
+        "building": "Budynki",
+        "current": "Numer (aktywny)",
+        "extension": "Rozbudowa",
+        "maximum": "Maksymalna możliwa",
+        "title": "Wszystkie {typy}",
```
Comment (jxn-30) [inline src/modules/dashboard/i18n/pl_PL.json:17]: `{typy}` → `{type}` (it is a variable that will be replaced)

File: src/modules/messageTemplates/i18n/pl_PL.root.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+    "description": "Napisz własne szablony wiadomości w prywatnych rozmowach.",
+    "name": "Szablony wiadomości",
+    "settings": {
+        "name": "Nazwa",
+        "subject": "Podmiot",
+        "template": "Szablon (zmienne (zobacz wiki): {{Nazwa Użytkownika}}, {{dzisiaj[+n]}})",
```
Comment (jxn-30) [inline src/modules/messageTemplates/i18n/pl_PL.root.json:7]: `{{Nazwa Użytkownika}}` needs to stay `{{username}}` and `{{dzisiaj[+n]}}` needs to stay `{{today[+n]}}`

**Review by jxn-30 — CHANGES_REQUESTED** [review]

File: src/modules/dailyCreditsSummary/i18n/pl_PL.ts [hunk 4]
```diff
@@ -0,0 +1,106 @@
+export default {
+    amount: 'Amount',
```
Comment (Cr4zyc4k3) [inline src/modules/dailyCreditsSummary/i18n/pl_PL.ts:2]: Amount = Kwota?

File: src/modules/dailyCreditsSummary/i18n/pl_PL.ts [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+    charts: {
+        income: 'Dochód',
+        expenses: 'Wydatki',
+    },
+    categories: {
+        allianceMission: {
+            regex: /^\[Alliance\]/u,
```
Comment (Cr4zyc4k3) [inline src/modules/dailyCreditsSummary/i18n/pl_PL.ts:9]: Is an alliance also called alliance in the polish version?

File: src/modules/dailyCreditsSummary/i18n/pl_PL.ts (hunk 4 above)
Comment (Kemon22) [reply src/modules/dailyCreditsSummary/i18n/pl_PL.ts:2]: yes

Comment (Kemon22) [reply src/modules/dailyCreditsSummary/i18n/pl_PL.ts:2]: Kwota (money) or Ilość (how many/much)

File: src/modules/dailyCreditsSummary/i18n/pl_PL.ts (hunk 5 above)
Comment (Kemon22) [reply src/modules/dailyCreditsSummary/i18n/pl_PL.ts:9]: alliance=sojusz

File: src/modules/dailyCreditsSummary/i18n/pl_PL.ts (hunk 4 above)
Comment (Cr4zyc4k3) [reply src/modules/dailyCreditsSummary/i18n/pl_PL.ts:2]: Do you want to change it then or should it stay as "amount"?

File: src/modules/dailyCreditsSummary/i18n/pl_PL.ts (hunk 5 above)
Comment (Cr4zyc4k3) [reply src/modules/dailyCreditsSummary/i18n/pl_PL.ts:9]: As @jxn-30 mentioned: We'll leave it and work on it after the merge

File: src/modules/dailyCreditsSummary/i18n/pl_PL.ts (hunk 4 above)
Comment (Kemon22) [reply src/modules/dailyCreditsSummary/i18n/pl_PL.ts:2]: It should be changed

**Review by jxn-30 — APPROVED** [review]

**Review by Cr4zyc4k3 — APPROVED** [review]

**Pull request merged**